vtkext/private/module/vtkF3DEXRReader.cxx

```diff
@@ -212,6 +207,10 @@ void vtkF3DEXRReader::ExecuteDataWithInformation(vtkDataObject* output, vtkInfor
     }
     else
     {
+      if (!(this->InternalFileName))
+      {
+        throw std::invalid_argument("Not filename in EXR Reader when no Memory Buffer is set.");
```

Comment:
can this happen ?

Do you really need to throw here ? you could just error out ?

Reply:
What i mean is that we do not need to throw here.